src/messages/generic_api.py

```diff
@@ -32,14 +32,10 @@ def send_message(self, msg: str):
     def _send(self, msg: str, loot = False):
         msg = f"{Config().general['name']}: {msg}"
 
-        if loot:
```

Comment:
I simplified the logic here. Does the difference make sense to you? The walrus operator `:=` is great for this sort of scenario--if you haven't learned about it yet check it out.